scripts/bundler.js

```diff
@@ -42,7 +42,8 @@ async function build() {
 		return
 	}
 	console.log("minifying...")
-	const minified = Terser.minify(original)
+	// Terser's  “compress” option seems to degrade performance. So, disable it.
+	const minified = Terser.minify(original, {compress: false, mangle: true})
```

Comment:
Could you see if just disabling `reduce_funcs` helps, and keeping the rest of compression on? https://github.com/terser/terser/issues/1305 among others are related.

Reply:
I tried the following setting where only `reduce_funcs` is false, but the output result seems to be the same as in v2.2.11...

```
const minified = Terser.minify(original, {compress: {reduce_funcs: false}, mangle: true})
```

Reply:
It seems that a newer version of terser works well with `reduce_funcs: false` and produces output that is both small file size and performance. However, installing a newer version of terser seems to install packages that do not support node v18.